corehq/apps/change_feed/consumer/feed.py

```diff
@@ -101,6 +102,9 @@ def get_processed_offsets(self):
     def get_latest_offsets(self):
         return get_multi_topic_offset(self.topics)
 
+    def get_latest_offsets_json(self):
+        return json.loads(kafka_seq_to_str(self.get_latest_offsets()))
```

Comment:
If you moved `seq = {'{},{}'.format(*key): val for key, val in seq.items()}` into its own function in [pillowtop/models.py](https://github.com/dimagi/commcare-hq/blob/718bc0c1f5e8b2d98ddb5fe7016e23fd7b935747/corehq/ex-submodules/pillowtop/models.py#L23-L23), say, `seq_to_kafka_seq()`, then you wouldn't need to call `json.loads()` on something that's just been returned from `json.dumps()`.